src/librustc_mir/hair.rs

```diff
@@ -113,45 +113,128 @@ pub struct Expr<'tcx> {
 
 #[derive(Clone, Debug)]
 pub enum ExprKind<'tcx> {
```

Comment:
This enum has inflated from 40 lines to 120. Did its author perhaps intend it to fit on a single screen?